tests/load-tests/cluster_read_config_stage.yaml

```diff
@@ -0,0 +1,100 @@
+  monitoring_query: count(kube_pod_info)
+  monitoring_step: 15
+
+- name: measurements.cluster_running_pods_on_workers_count
+  monitoring_query: count(kube_pod_info * on(node) group_left(role) kube_node_role{role="worker"} and on(pod, namespace) (kube_pod_status_phase{job="kube-state-metrics", phase="Running"} > 0))
+  monitoring_step: 15
+
+- name: measurements.scheduler_pending_pods_count
+  monitoring_query: sum(scheduler_pending_pods)
+  monitoring_step: 15
+
+- name: measurements.tekton_tekton_pipelines_controller_workqueue_depth
+  monitoring_query: sum(tekton_pipelines_controller_workqueue_depth)
+  monitoring_step: 15
+
+- name: measurements.pipelinerun_duration_scheduled_seconds
+  monitoring_query: sum(pipelinerun_duration_scheduled_seconds_sum / pipelinerun_duration_scheduled_seconds_count)
+  monitoring_step: 15
+
+- name: measurements.tekton_pipelines_controller_running_taskruns_throttled_by_node
+  monitoring_query: sum(tekton_pipelines_controller_running_taskruns_throttled_by_node_count)
+  monitoring_step: 15
+
+- name: measurements.tekton_pipelines_controller_running_taskruns_throttled_by_quota
+  monitoring_query: sum(tekton_pipelines_controller_running_taskruns_throttled_by_quota_count)
+  monitoring_step: 15
+
+- name: measurements.etcd_request_duration_seconds_average
+  monitoring_query: sum(rate(etcd_request_duration_seconds_sum{}[5m])) / sum(rate(etcd_request_duration_seconds_count[5m]))
+  monitoring_step: 15
+
+- name: measurements.cluster_network_bytes_total
+  monitoring_query: sum(irate(container_network_receive_bytes_total{cluster="",namespace=~".*"}[5m])) + sum(irate(container_network_transmit_bytes_total{cluster="",namespace=~".*"}[5m]))
+  monitoring_step: 15
+
+- name: measurements.cluster_network_receive_bytes_total
+  monitoring_query: sum(irate(container_network_receive_bytes_total{cluster="",namespace=~".*"}[5m]))
+  monitoring_step: 15
+
+- name: measurements.cluster_network_transmit_bytes_total
+  monitoring_query: sum(irate(container_network_transmit_bytes_total{cluster="",namespace=~".*"}[5m]))
+  monitoring_step: 15
+
+- name: measurements.node_disk_io_time_seconds_total
+  monitoring_query: sum(irate(node_disk_io_time_seconds_total{cluster="",namespace=~".*"}[5m]))
+  monitoring_step: 15
+
+# Interesting CI environment variables
+{% for var in [
+  'BUILD_ID',
+  'HOSTNAME',
+  'JOB_NAME',
+  'OPENSHIFT_API',
+  'PROW_JOB_ID',
+  'PULL_BASE_REF',
+  'PULL_BASE_SHA',
+  'PULL_HEAD_REF',
+  'PULL_NUMBER',
+  'PULL_PULL_SHA',
+  'PULL_REFS',
+  'REPO_NAME',
+  'REPO_OWNER',
+  'SCENARIO',
+] %}
+- name: metadata.env.{{ var }}
+  env_variable: {{ var }}
+{% endfor %}
+
+- name: metadata.scenario
+  command: if [ -r /usr/local/ci-secrets/redhat-appstudio-load-test/load-test-scenario.${SCENARIO} ]; then cat /usr/local/ci-secrets/redhat-appstudio-load-test/load-test-scenario.${SCENARIO} | sed 's/\\ /,/g' | sed 's/[^ ]* \([^= ]*\)=\([^= ]*\)/"\1":"\2",/g' | sed 's/\(.*\),$/{\1}/g'; else echo '{}'; fi
```

Comment:
@naftalysh This change to `metadata.scenario` command looks nice. Please apply it to the main metric config yaml (`./test/load-tests/cluster_read_config.yaml` as well (as you did before). Thanks!

Reply:
Sure